docs/can-guides/experiment/technology/routing.md

```diff
@@ -216,10 +216,10 @@ In most applications, [can-route] is connected to a property on the top-level co
 building `<my-app>` and connecting it
 to [can-route]. This is usually done in five steps:
 
-1. Define the top-level component’s view-model (sometimes called _application view-model_).
+1. Define the top-level component’s view-model (a [can-define/map/map DefineMap] sometimes called _application view-model_).
```

Comment:
The VM can be a can-map, or can-observe, or…

Was there something confusing about not mentioning the type here?

Reply:
I think it's always good to add links though, don't you think? I think the wording could be changed to be more non-specific.  *such as a DefineMap*

Reply:
Is it most likely to be a _DefineMap_? I don't think there's any harm in pointing out what it's most likely going to be. If so, instead of _such as a DefineMap_, it could read _typically a DefineMap_. Otherwise should probably say _an Object_ or just not change this sentence.

Reply:
I made a small edit to say “typically.” Thanks!